### PR TITLE
Repfix

### PR DESCRIPTION
**Pull request by jzenoz**

### CODE REVIEW

**Comment by jzenoz** [comment]
For whatever reason when who is filled in tradelog the value is rounded up to the nearest 100... but only for 664* ID users...
664536215265214475 is displayed as: 
664536215265214500
but doesn't affect the functionality with keep_bot_in_check or status...

File: traderep/traderep.py [hunk 1]
```diff
@@ -144,182 +148,13 @@ def __init__(self, bot):
...
-                            row[0], trade_num
-                        ))
-                cur.execute("INSERT INTO tradelog (logtime, who, tradenum, what) values ('now', ?, ?, ?)",
-                            (ctx.message.author.id, trade_num, "Author derepped {} for  trade {}".format(
-                                trade_who, trade_num
-                            )))
-            else:
-                await ctx.send("I didn't find a trade matching that description which involved you")
-        else:
-            await ctx.send("I didn't find a trade matching that description which involved you")
```
Comment (jzenoz) [inline traderep/traderep.py]: derep and rep are combined to the same code base repmod below

File: traderep/traderep.py [hunk 2]
```diff
@@ -385,38 +385,34 @@ def __init__(self, bot):
...
             try:
                 cur.execute("INSERT INTO tradelog (logtime, who, tradenum, what) values (DateTime('now'), ?, ?, ?)",
-                            (ctx.message.author.id, trade_num, "Author {} {} for trade {}".format(mod_word, trade_who, trade_num)))
+                            (ctx.message.author.id, trade_num, "Author {} {} for trade {}".format(mod_word, partner.id, trade_num)))
             except sqlite3.OperationalError as e:
                 await ctx.send("There's a problem right now: {}".format(e))
                 return
             if self.bot.user.id == partner.id:
-                await keep_bot_in_check(ctx, trade_num, cur, self.bot.user.id)
+                await keep_bot_in_check(ctx, trade_num, cur, partner.id)
```
Comment (jzenoz) [inline traderep/traderep.py]: these changes don't do anything, just aligning

File: traderep/traderep.py [hunk 3]
```diff
@@ -353,7 +353,7 @@ def __init__(self, bot):
         return
     # Find the open trade number
     cur.execute("SELECT tp.tradenum, partner from tradeperson tp join trade t on t.tradenum = tp.tradenum"
-                " where tp.tradenum = ? and person = ? and (t.status = 1 or t.status is null)"
+                " where tp.tradenum = ? and person = ? and (t.status is null or tp.rep is null)"
```
Comment (jzenoz) [inline traderep/traderep.py]: This is the current bug fix

File: traderep/traderep.py [hunk 4]
```diff
@@ -385,38 +385,34 @@ def __init__(self, bot):
...
             else:
-                if did_close:
-                    await ctx.send("You closed your end of trade {} and {} {} for it. It's their turn to rep you..."
-                                   .format(trade_num, mod_word, partner.mention))
-                else:
-                    await ctx.send("You {} {} for trade {} and that should mean all reps complete...".format(
-                        mod_word, partner.mention, trade_num
-                    ))
+                await ctx.send("get_user_by_id failed, yell at someone! {} {} {}"
+                               .format(ctx, trade_who, self))
```
Comment (jzenoz) [inline traderep/traderep.py]: This was never reached in my testing.

Comment (AardWolf) [reply traderep/traderep.py]: Yeah, logic changed early in development. I think from feedback from you.

**Pull request merged**